ui/components/app/incoming-trasaction-toggle/NetworkToggle.js

```diff
@@ -0,0 +1,92 @@
+import React, { useContext } from 'react';
```

Comment:
Question: Why do we have NetworkToggle.js and NetworkToggle.tsx ? Both files look same to me. Also, naming should be kebab-case